server/contracts/test/BlockChat.test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require("assert");
+
+
+const BlockChat = artifacts.require("BlockChat");
+
+contract("BlockChat", accounts => {
+    let token
+
+    before(async () => {
+      token = await BlockChat.deployed()
+    })
+  it("should be deployed", () =>{
+    const address = token.address
+    assert.notEqual(address, 0x0)
+    assert.notEqual(address, '')
+    assert.notEqual(address, null)
+    assert.notEqual(address, undefined)
+  })
+  it("should send a message", async () =>{
+    const sendMessage = await token.sendMessage('Hello Blockchain!', '1')
+    assert.notEqual(sendMessage, undefined)
```

Comment:
Same as above, if you change this to `assert.ok(sendMessage)` it will ensure that the message is not null, empty or undefined 🤯